CHANGELOG.md

```diff
@@ -3,6 +3,10 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## [4.4.3] - TBD
```

Comment:
```suggestion
## [4.5.0] - TBD
```
(since you're adding a feature here not fixing a bug IMHO - NABD either way though)

Reply:
Don't you also want to mention the other added variable for controlling the role session duration?

Reply:
That feature was already there from a previous change.  I just added the documentation while I was making my change.